src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -267,6 +267,18 @@ categories :
       description : >-
         Specifies the number of buffers for receiving active messages.
 
+    - name        : MPIR_CVAR_CH4_OFI_RMA_FLUSH_INTERVAL
+      category    : CH4_OFI
+      type        : int
+      default     : 100
+      class       : device
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_LOCAL
+      description : >-
+        Specifies the interval of manual flush for RMA operations. This is effective
+        only when the underlying OFI provider does manual data progress. It it supports
```

Comment:
Can we have a way to disable the manual progress (e.g., using CVAR)? And it will be good if we could explain in the CVAR that this is used only for noncontiguous RMA.

`MPIR_CVAR_CH4_OFI_RMA_FLUSH_INTERVAL` reads like force completion. Maybe `PROGRESS_INTERNAL` is more suitable ?

Reply:
There is already a CVAR to pick manual/automatic progress for OFI (`MPIDI_OFI_ENABLE_DATA_AUTO_PROGRESS`) unless you mean a CVAR specifically to disable this optimization. We could add a special case for `-1` where it wouldn't enable this (and not need another CVAR).